notesapi/v1/views.py

```diff
@@ -64,7 +71,7 @@ def get_from_es(self, *args, **kwargs):  # pylint: disable=unused-argument
             opts = {
                 'pre_tags': ['<{tag}{klass_str}>'.format(
                     tag=tag,
-                    klass_str=' class="{}"'.format(klass) if klass else ''
+                    klass_str=' class=\\"{}\\"'.format(klass) if klass else ''
```

Comment:
The tags are stored in the database as a single JSON string array in string from. For example, `tags` might be `["foo", "bar"]` as a string. If we insert the highlight tag around `foo` without a `\` then we could get `["<span class="highlight">foo</span>", "bar"]`. This is no longer valid JSON. Instead the quotes in the injected tag are escaped, so we get `["<span class \"=highlight\">foo</span>", "bar"]` which can be deserialized successfully.